vtex-apps/store-sitemap
# New sitemap

# What is the purpose of this PR?

This PR creates a new sitemap using routes from `vtex.rewriter`

# How to test it 

Two workspaces to test:
[Storecomponents](https://newsitemap--storecomponents.myvtex.com/sitemap.xml)
[PowerPlanet](https://newsitemap--powerplanetonline.myvtex.com/sitemap.xml)

To try it for yourself you should:

1. Have some routes in rewriter, you can navigate in your workspace to save them or re-index all routes by doing a: `GET https://app.io.vtex.com/vtex.broadcaster-worker/v0/storecomponents/newsitemap/index-routes` (with the VtexIdclientAutCookie Cookie)

2. Generate the sitemap by doing the following request: 
`GET https://app.io.vtex.com/vtex.store-sitemap/v1/storecomponents/fox/generate-sitemap`

3. Access the sitemap with the path: `/sitemap.xml`
